package.json

```diff
@@ -13,32 +13,39 @@
     "lint": "tslint --project ."
   },
   "dependencies": {
-    "@next/bundle-analyzer": "^10.0.4",
-    "@types/node-fetch": "^2.5.7",
-    "bootstrap": "^4.5.3",
-    "classnames": "^2.2.6",
-    "jest": "^26.0.19",
-    "lodash": "^4.17.20",
-    "next": "10.0.7",
+    "@next/bundle-analyzer": "^11.1.0",
+    "@types/node-fetch": "^2.5.12",
+    "bootstrap": "^5.1.0",
```

Comment:
Pretty big jump from Bootstrap 4 to Bootstrap 5 - as well as `react-sortable-hoc` and `next`. I would leave these for a later time when I have time to go over the changelogs and see what changed 😄

Reply:
I updated these dependencies because I was having a real hard time getting the react testing framework to work. It's working now so I should be able to remove the major version upgrades from this branch.